webrtc.html

```diff
@@ -1156,7 +1156,7 @@
             by the <a>ICE Agent</a> since the offer or answer was created. If the
            <code>RTCPeerConnection</code> is in the stable state, the value is null.
             This attribute is updated by <code><a href=
-            "#dom-peerconnection-setlocaldescription">setLocalDescription()</a></code>.
+            "#dom-peerconnection-setremotedescription">setRemoteDescription()</a></code>.
```

Comment:
I think this is wrong, the correct place for this seems to be line 1262, and there it already says setRemoteDescription